src/towncrier/create.py

```diff
@@ -22,12 +22,17 @@
     default=False,
     help="Open an editor for writing the newsfragment content.",
 )  # TODO: default should be true
+@click.option(
+    "-m", "--message",
+    type=str,
+    help="Set message in new fragment."
+)
```

Comment:
I am OK with `message`.

Just a minor suggest to consider, but feel free to discard it.

```suggestion
@click.option(
    "-c", "--content",
    type=str,
    help="Sets the content of the new fragment."
)
```